src/dapps/ManageENS/ManageENS.vue

```diff
@@ -607,11 +619,13 @@ export default {
           ResolverAbi,
           this.publicResolverAddress
         );
+        this.fetchTxtRecords(publicResolverContract);
         this.resolverMultiCoinSupport = await publicResolverContract.methods
           .supportsInterface(MULTICOIN_SUPPORT_INTERFACE)
           .call();
       } catch (e) {
         this.resolverMultiCoinSupport = false;
+        this.resolverTxtSupport = false;
```

Comment:
You have to check these separately, right now if one fails other one fails as well